packages/cli/cli.js

```diff
@@ -40,6 +41,7 @@ program.register('help', help.toStdout)
 program.register('help db', async (args) => runDB(['help', ...args]))
 program.register('help service', async (args) => runService(['help', ...args]))
 program.register({ command: 'login', strict: true }, login)
+program.register('gh', gh)
```

Comment:
IMHO, I would use the `cloud` subcommand
cc @simonplend

Reply:
I think we should rename this to `actions`, but I'm bikeshedding.